pkg/kernel/kernel_test.go

```diff
@@ -12,3 +15,23 @@ func TestGetKernelVersion(t *testing.T) {
 
 	println(version.String())
 }
+
+// Benchmark result for below two methods to execute `uname` command in GetKernelVersion().
+
+// BenchmarkGetUnameByUnix-4      	  200000	     10584 ns/op
+// BenchmarkGetUnameByExecRun-4   	     200	   6255530 ns/op
+
+// Benchmark for executing `uname` by raw unix system call
```

Comment:
Just asking, how long this two Benchmark case execute?

Reply:
![image](https://user-images.githubusercontent.com/19202395/48308550-d3762c80-e5a2-11e8-930c-44e2efd27ae1.png)
the result shows 4.135s